documentation/en/default-lotus-config.toml

```diff
@@ -241,6 +241,16 @@
   # env var: LOTUS_FEVM_ETHTRACEFILTERMAXRESULTS
   #EthTraceFilterMaxResults = 500
 
+  # EthBlkCacheSize specifies the size of the cache used for caching Ethereum blocks.
+  # This cache enhances the performance of the eth_getBlockByHash RPC call by minimizing the need to access chain state for
+  # recently requested blocks that are already cached.
```

Comment:
I would consider having a separate "recency" cache for the latest N blocks (just to _ensure_ that those requests are very fast). I'm thinking, e.g., this could be tied into the gateway's lookback limit so that all blocks a user _can_ request are also cached.

But this isn't critical and should definitely happen in a followup PR (if ever).